app/default/transforms.conf

```
@@ -1,9 +1,35 @@
-# Drop events for successful __gtg and __health requests
-# these logs don't count towards our injest license usage, index time
-[heroku_requests_doubleunderscore_null]
+# Drop events for successful __gtg and __health requests these logs don't count towards our injest license usage, index time.
+[heroku_requests_null_queue]
 REGEX = router\s-\s.+path="\/__(gtg|health)"\s.+status=200
 DEST_KEY = queue
 FORMAT = nullQueue
 
+# Various types of log messages for Heroku log drains. See https://devcenter.heroku.com/articles/logging#runtime-logs for more information.
+[heroku_app_source_type]
+REGEX = ^\S+\sapp\s
+FORMAT = sourcetype::heroku_app
+DEST_KEY = MetaData:Sourcetype
```

Apparently this is required?

```suggestion
DEST_KEY = MetaData:Sourcetype
WRITE_META = true
```

---

But locally it all seems to worth without this 🤷